tests/test_main.py

```diff
@@ -128,11 +159,11 @@ def test_format_output_auto_expand():
     )
     table = [
         "Title",
-        "+---------+---------+",
-        "| head1   | head2   |",
-        "|---------+---------|",
-        "| abc     | def     |",
-        "+---------+---------+",
+        "+-------+-------+",
```

Comment:
Not sure why this changed - changes in tabulate? Seems unrelated to the new settings.